client/src/Components/shared/Navbar.js

```diff
@@ -20,6 +21,17 @@ const useStyles = makeStyles((theme) => ({
   toolbarLink: {
     padding: theme.spacing(1),
     flexShrink: 0,
+    marginRight: '4%'
```

Comment:
be very careful with using %s in CSS - make sure it looks good on all screen sizes!